docs/config/aws_template.md

```diff
@@ -0,0 +1,441 @@
+# Deploying to AWS
+
+!!! danger
+
+    This deployment can only be run by AWS administrators, so we assume the user has a basic understanding of code
+    development, Docker containers, and the AWS cloud.
+
+## Overview
+
+The end result of this guide is a functioning pipeline in AWS that allows you to process data on-the-fly as it enters
```

Comment:
The goal of this guide is to help you create a functioning pipeline in AWS that allows you...

Reply:
Yep, that sounds better. Changed it